specs/deposits.md

```diff
@@ -54,6 +55,8 @@ fields (rlp encoded in the order they appear here):
 
 [EIP-2718]: https://eips.ethereum.org/EIPS/eip-2718
 
+- `uint64 blockHeight`: the block-height of the L2 block
+- `uint64 transactionIndex`: the transaction-index within the L2 transactions list
 - `address from`: The address of the sender account.
```

Comment:
Maybe make explicit what is from the deposit itself and what is generated by the rollup node when translating the transaction.

Reply:
That's already explicit in the `rollup-node.md` derivation process